NEWS.md

```diff
@@ -11,10 +11,16 @@
 
 ### Changes in pre-built images
 
-- cuda version tags, like `rocker/cuda:cuda10.1`, `rocker/cuda:4.2.1-cuda10.1` are obsolete for R 4.2.2 or later images.
-  Only the R version tags like `rocker/cuda:4.2.2`, `rocker/cuda:latest` will remain.
+- cuda version tags, like `rocker/cuda:cuda10.1`, `rocker/ml:4.2.1-cuda10.1` are obsolete for R 4.2.2 or later images.
+  Only the R version tags like `rocker/cuda:4.2.2`, `rocker/ml-verse:latest` will remain.
+  `rocker/cuda` R >= 4.2.2 corresponds to `rocker/cuda:cuda11.1` before R.4.2.1 and
+  uses `nvidia/cuda` as the base image.
+  Therefore, the image size is significantly different from `rocker/cuda` R <= 4.2.1,
```

Comment:
I think we should say "significantly larger" rather than "different".

Reply:
also, maybe we should comment specifically that it uses `nvidia/cuda` devel images (tag `cudnn8-devel-ubuntu22.04`) -- arguably we might get away with `runtime`, I'm not sure we need devel and could save some significant space.  (On the plus side users compiling against cuda libs might appreciate this?)

Reply:
Thanks for the review. I have made the changes.

BTW, `nvidia/cuda:11.8.0-cudnn8-runtime-ubuntu22.04` seems to be half the size of `nvidia/cuda:11.8.0-cudnn8-devel-ubuntu22.04`.
https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=22.04

I don't usually use cuda so I don't know what is different about these images and how they benefit the user.
It may be worth creating an issue about changing the base image.